README.md

```diff
@@ -4,24 +4,29 @@
 
 [![Build Status](https://travis-ci.org/redhat-developer/odo.svg?branch=master)](https://travis-ci.org/redhat-developer/odo) [![codecov](https://codecov.io/gh/redhat-developer/odo/branch/master/graph/badge.svg)](https://codecov.io/gh/redhat-developer/odo) [![CircleCI](https://circleci.com/gh/redhat-developer/odo/tree/master.svg?style=svg)](https://circleci.com/gh/redhat-developer/odo/tree/master) [![mattermost](/docs/img/mattermost.svg)](https://chat.openshift.io/developers/channels/odo)
 
-A fast iterative tool for deploying your source code straight to OpenShift.
+OpenShift Do (odo) is a fast, iterative, and opinionated CLI tool for developers who write, build, and deploy applications on OpenShift.
 
-## Features
+Existing tools such as `oc` are more operations-focused and require a deep-understanding of Kubernetes and OpenShift concepts. OpenShift Do abstracts away complex Kubernetes and OpenShift concepts, thus allowing developers to focus on what's most important to them: code.
 
-- Designed for fast, iterative development cycles
-- 100% client based. No server required within your OpenShift cluster for deployment
-- Supports multiple languages and frameworks such as Node.js, Java, Ruby, Perl, PHP and Python
-- Detect changes to your local code and deploy automatically with `odo watch`
-- List all available components and services from your OpenShift cluster
+## Key features
+OpenShift Do is designed to be simple and concise with the following key features:
+
+- Its Syntax and design are centered around concepts familiar to developers, such as project, application, and component.
+- It is completely client based. No server is required within the OpenShift cluster for deployment.
+- It supports multiple languages and frameworks such as Node.js, Java, Ruby, Perl, PHP, and Python.
```

Comment:
IMO, It's / its / it is a bit repetitive. Should we remove it all together, does:

`Supports multiple languages and frameworks such as Node.js, Java, Ruby, Perl, PHP, and Python.`

sound better than

`It upports multiple languages and frameworks such as Node.js, Java, Ruby, Perl, PHP, and Python.` ?